prefab/prefab-publishing/.clang-format

```diff
@@ -1 +0,0 @@
-BasedOnStyle: Google
```

Comment:
Why? This is useful.

Reply:
no, it is command line parameter, right.

Reply:
We have command line users. It's also usable by Android Studio: https://www.jetbrains.com/help/clion/clangformat-as-alternative-formatter.html#config-files

Reply:
out of ~200 samples, no one checks in this file except for ios samples. Would be good to keep the tradition aligned with the rest of the Android samples.

Reply:
Yeah, it would be good to add it to all of the other samples. Could move this to the top level instead if you prefer.